### PR TITLE
Istio  - Support type loadbalancer in notes

### PR DESCRIPTION
**Pull request by lachie83**
Merge this after #1181. It will need a rebase

### CODE REVIEW

**Comment by unguiculus** [comment]
Needs a rebase.

**Comment by lachie83** [comment]
@unguiculus rebased

File: incubator/istio/templates/NOTES.txt [hunk 1]
```diff
@@ -3,19 +3,40 @@ Verify all Pods are running
...
-  export POD_NAME=$(kubectl get pods --namespace {{ .Release.Namespace }} -l "app={{ $serviceName }}-servicegraph" -o jsonpath="{.items[0].metadata.name}")
+  NOTE: It may take a few minutes for the LoadBalancer IP to be available.
+        Watch the status with: 'kubectl get svc --namespace {{ .Release.Namespace }} -w {{ $serviceName }}-servicegraph'
+
+  export SERVICE_IP=$(kubectl get svc --namespace {{ .Release.Namespace }} {{ $serviceName }}-servicegraph -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
+  echo http://$SERVICE_IP:{{ .Values.addons.servicegraph.service.externalPort }}/dotviz
+{{- else }}
+Verifying the ServiceGraph service
+
+  export POD_NAME=$(kubectl get pods --namespace {{ .Release.Namespace }} -l "app={{ $serviceName }}-serviegraph" -o jsonpath="{.items[0].metadata.name}")
```
Comment (unguiculus) [inline incubator/istio/templates/NOTES.txt]: Typo: serviegraph -> servicegraph

**Review by unguiculus — CHANGES_REQUESTED** [review]

File: incubator/istio/templates/NOTES.txt (hunk 1 above)
Comment (lachie83) [reply incubator/istio/templates/NOTES.txt]: Will fix

**Review by unguiculus — APPROVED** [review]

**Pull request merged**